IFComp/lib/IFComp/Controller/Play.pm

```diff
@@ -68,7 +68,7 @@ sub download :Chained('fetch_entry') :Args(0) {
 
     my $entry = $c->stash->{ entry };
     my $filename = $entry->main_file->basename;
-    if ( $entry->platform eq 'html' ) {
+    if ( $filename =~ /\.html?$/i ) {
```

Comment:
Change this test since the html platform was removed and no longer distinguishes between single-file and multiple html files in a zip